src/templates/app/cashier/index.jsx

```diff
@@ -85,9 +104,9 @@ const Cashier = () => (
                         <img className='responsive' id='payment_agent_icon' src={it.url_for('images/pages/cashier/payment-agents.svg')} />
                     </a>
                 </div>
-                <div className='gr-5 gr-12-m'>
+                <div className='gr-6 gr-12-m'>
                     <span>{it.L('For e-wallets or local currencies not supported by [_1].', it.website_name)}</span>
-                    <p className='faded'>{it.L('Note: Withdrawal via payment agent is available only if you deposit exclusively via payment agent')}</p>
+                    <CashierNote text={it.L('Withdrawal via payment agent is available only if you deposit exclusively via payment agent.')} />
```

Comment:
doesn't that mean this will be hidden for all except those four countries too? do we want that?

Reply:
let me check

Reply:
@negarn we hide this except for those 4 countries too